src/Core/System.php

```diff
@@ -161,6 +161,18 @@ public static function jsonExit($x, $content_type = 'application/json') {
 		killme();
 	}
 
+	/**
+	 * Generates a random string in the UUID format
+	 *
+	 * @param bool|string  $prefix   A given prefix (default is empty)
+	 * @return string a generated UUID
+	 */
+	public static function UUID($prefix = '')
```

Comment:
can you please rename the method to `createUUID()`. So that it is easier for other people to see what this method is about